nimbus/src/client/http_client.rs

```diff
@@ -13,10 +13,11 @@
 //!
 //! But the simple subset implemented here meets our needs for now.
 
+use std::str::FromStr;
```

Comment:
If you use `.parse` as I suggest in a separate comment, I don't think you'll need this import.